pcweb/pages/index.py

```diff
@@ -330,9 +330,8 @@ def request_buttons() -> rx.Component:
         rx.button(
             "Bugs",
             color="#2BCEEA",
-            weight="Medium",
-            height="22px",
-            width="138px",
+            height="26px",
```

Comment:
Since this is in a stack, can we not set the width and just put a spacer at the end?